classes/module/Module.php

```diff
@@ -982,8 +986,23 @@ public function disable($force_all = false)
 
         // Disable module for all shops
         $sql = 'DELETE FROM `' . _DB_PREFIX_ . 'module_shop` WHERE `id_module` = ' . (int) $this->id . ' ' . ((!$force_all) ? ' AND `id_shop` IN(' . implode(', ', Shop::getContextListShopID()) . ')' : '');
+        $resultDeletionQuery = Db::getInstance()->execute($sql);
```

Comment:
```suggestion
        $resultDeletionQuery = Db::getInstance()->delete('module_shop', '`id_module` = ' . (int) $this->id );
```

With the remove of the `$sql`